classes/mender-sdimg.bbclass

```diff
@@ -36,7 +36,19 @@ SDIMG_BOOT_PART_SIZE_MB ?= "16"
 #
 # 8MB alignment is a safe setting that might waste some space if the
 # erase block is smaller.
-SDIMG_PARTITION_ALIGNMENT_MB ?= "8"
+MENDER_PARTITION_ALIGNMENT_MB ?= "8"
+
+python() {
+    deprecated_vars = ['SDIMG_DATA_PART_DIR', 'SDIMG_DATA_PART_SIZE_MB',
+                       'SDIMG_BOOT_PART_SIZE_MB', 'SDIMG_PARTITION_ALIGNMENT_MB']
+    for varname in deprecated_vars:
+        cur = d.getVar(varname, True)
+        if cur:
+            newvarname = varname.replace('SDIMG_', 'MENDER_')
+            bb.warn('Detected use of deprecated var %s, reassigning to %s' % (varname, newvarname))
+            d.setVar(newvarname, cur)
+            d.setVar(varname, '')
+}
```

Comment:
I would vote for doing hard error here instead. We are still in the early stages and it would be nice if we don't drag with us old deprecated variables. Later on, with more widespread adoption I'm thinking warning might be better. Do you agree?